package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@aws-sdk/client-kms": "^3.485.0",
+    "@emurgo/cardano-serialization-lib-nodejs": "11.5.0",
```

Comment:
can remove. not a monorepo dependency, it's only directly referenced in the example